docs-v2/integrations/all/gumroad.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Gumroad
+sidebarTitle: Gumroad
+---
+
+API Configuration: [`asana`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Feature                                                                          | Status                          |
+| -------------------------------------------------------------------------------- | ------------------------------- |
+| [Auth (OAuth)](/nango-auth)                                                      | ✅                              |
+| [Sync](/nango-sync)                                                              | ✅                              |
+| [Nango Proxy](/nango-sync/guides/nango-proxy)                          | ✅                              |
```

Comment:
It doesn't look like the proxy is supported, so the checkmark should be changed to "🚫 (time to contribute: &lt;1h)"

Reply:
Thanks, will make all the changes.